packages/react-native-worklets/package.json

```
@@ -66,6 +66,8 @@
     "android",
     "apple",
     "Common",
+    "scripts",
+    "plugin",
```

Only take the output file, like in Reanimated's `package.json`.

```suggestion
    "plugin/index.js",
```